indico/modules/events/requests/templates/event_request_details.html

```diff
@@ -76,15 +83,8 @@
     {% endif %}
 
     {% call form_footer(form) %}
-        <input class="i-button big highlight" type="submit" value="
-            {%- if not req -%}
-                {%- trans -%}Send request{%- endtrans -%}
-            {%- elif req.can_be_modified -%}
-                {%- trans -%}Modify request{%- endtrans -%}
-            {%- else -%}
-                {%- trans -%}Send new request{%- endtrans -%}
-            {%- endif -%}
-        ">
+        <input id="send-request" class="i-button big highlight" type="submit"
+               value="{% trans %}{{ button_title }}{% endtrans %}" data-disabled-until-change>
```

Comment:
As @ThiefMaster mentioned, you cannot do the translation like this. The content of variables is not translated inside the `{% trans %}` block.